src/main/java/com/sprints/model/Sprint.java

```diff
@@ -1,5 +1,24 @@
 package com.sprints.model;
 
-public class Sprint {
+import java.time.LocalDateTime;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import lombok.Data;
 
-}
+@Document(collection="sprints")
+@Data
+public class Sprint {
+	@Id
+	String id;
+	String technology;
+	boolean active;
+	boolean is_backlog;
+	
+	@DateTimeFormat(pattern="MM/dd/yyyy")
```

Comment:
If you only need to store Date use LocalDate instead of LocalDateTime.

Reply:
Done